sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -770,13 +770,13 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
       val parquetDF = spark.read.parquet(parquetDir)
       assert(parquetDF.inputFiles.nonEmpty)
 
-      val jsonDir = new File(dir, "json").getCanonicalPath
-      df.write.json(jsonDir)
-      val jsonDF = spark.read.json(jsonDir)
-      assert(parquetDF.inputFiles.nonEmpty)
+      val csvDir = new File(dir, "csv").getCanonicalPath
```

Comment:
Not sure if we should test enable/disable file source v2 for all the test cases, or just test with the default configuration.

Reply:
For this case, I think we should test both v1 and v2.

Reply:
BTW why we change from json to csv?

Reply:
> BTW why we change from json to csv?

Because Json V2 PR is still in review...:)